docs/content/usage/packages/container.en-us.md

```diff
@@ -23,6 +23,23 @@ The container registry follows the OCI specs and supports all compatible images
 To work with the Container registry, you can use the tools for your specific image type.
 The following examples use the `docker` client.
 
+### Reverse Proxy / URL Prefix
+
+The container registry uses a fixed url path `/v2` which can't be changed.
+Even if you deploy Gitea with a prefix, `/v2` will be used by the `docker` client.
+Therefore you may need to add an additional route to your reverse proxy configuration.
+
+Example using nginx:
+```
+location /gitea/ { ## normal Gitea route
+	proxy_pass http://gitea:3000/;
```

Comment:
It breaks `%2F` and causes various bugs.

* #23711

<details>

![image](https://github.com/go-gitea/gitea/assets/2114189/403e5b2c-3d4c-49fb-80c4-af7a5e7e98b9)

</details>

Reply:
What would be the suggested method?

Reply:
All have been written in the comment above.

Reply:
> All have been written in the comment above.

Thank you, I implemented it and it appears to be working great